src/data/markdown/docs/02 javascript api/13 k6-redis.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "k6/experimental/redis"
+excerpt: "k6 Redis experimental API"
+---
+
+<Blockquote mod='info'>
+
+#### This module is experimental, use at your own risk
+
+**This module is experimental**. It is not guaranteed to be stable, and may change without notice. **Use at your own risk**.
+While will we intend to keep this module as simple and stable as possible, we may need to add more features or introduce breaking changes in the future, and won't refrain from doing so until we release this module as a "stable" one. Please kindly provide us your user feedback, and open an issue or pull request if you have any suggestions.
+</Blockquote>
+
+The [Redis](https://redis.io/) module for [k6](https://github.com/grafana/k6) provides a client library allowing users to interact with Redis, directly from their k6 scripts. It was built with the objective to enable users to either load test their Redis infrastructure, or use Redis as a data store backing their load tests' script logic.
```

Comment:
```suggestion
The [Redis](https://redis.io/) module for [k6](https://github.com/grafana/k6) provides a client library to interact with Redis, directly from their k6 scripts.
Its objective is to let users to either load test their Redis infrastructure, or use Redis as a data store for their test-script logic.
```

Reply:
I don't have a strong reason to remove "load". I did it for a few reasons:
- k6 could be for general performance testing, too
- it shortens the sentence
- I think style guidelines would say it needed to be "load-test-script logic", and while this would be very pleasing to me, I think it is too fussy for our docs.

Reply:
The first sentence in your suggestion should drop the "their" from "directly from their k6 scripts". As the users it referred to is now gone 😄